src/app/CommandHandlerInterface.h

```diff
@@ -22,12 +22,16 @@
 #include <app/ConcreteClusterPath.h>
 #include <app/ConcreteCommandPath.h>
 #include <app/data-model/Decode.h>
-#include <app/data-model/List.h> // So we can encode lists
+#include <app/data-model/List.h>         // So we can encode lists
+#include <app/data-model/MetadataList.h> // So we can get filled buffers
 #include <lib/core/DataModelTypes.h>
 #include <lib/support/Iterators.h>
 
 namespace chip {
 namespace app {
+namespace DataModel {
+struct AcceptedCommandEntry;
```

Comment:
This seems to show dependency issues. Please include the header and do not forward-declare.